### DISCUSSION TITLE
EDK2 and Visual Studio 2022

### DISCUSSION THREAD

**Discussion by kowalski100** [tianocore/edk2#6309] [General]
Hi, 

The `edksetup.bat` `edk2-stable202408` tag has the following line:
if /I "%1"=="VS2022" shift

Which seems like the EDK2 supports Visual studio v2022 as build toolchain. but The README only mentions VS2019. So my question is: Can the EDK latest stable tag be built with VS2022 or is it tested with it?

**Comment by os-d** [comment]
@kowalski100 , yes, edk2 can be built with VS2022. The README and CI pipelines need to be updated. Someone had put up a PR to do so, but closed it. If you want to put up a PR for the same, please do. It has just been a matter of having the time to make the switch.
